### PR TITLE
fix: removed screenshot swap checks in favour of text check and tidied up services page objects and tests

### PR DESCRIPTION
**Pull request by ggilchrist-ledger**
### 📝 Description
NOTE: this PR is primarily to remove the screenshot which was failing due to flaky amounts in the swap verify drawer. We're now just checking the text and that the amount matches up to the first 3 decimals which are consistent as far as we can see. This issue should be fixed as the user could be seeing the wrong amount in the verify screen. Also the functions that generate the formatted amounts should have unit tests given it is a large, complex function and it is used in many places in LLD and LLM.

- Removed screenshot swap checks in favour of text check
- Tidied up services page objects and tests
### ❓ Context

- **Impacted projects**: `` 
- **Linked resource(s)**: `` 

### ✅ Checklist

- [x] **Test coverage** 
- [x] **Atomic delivery** 
- [x] **No breaking changes** 

### 🚀 Expectations to reach

_Please make sure you follow these [**Important Steps**](https://github.com/LedgerHQ/ledger-live/blob/develop/CONTRIBUTING.md#important-steps)._

_Pull Requests must pass the CI and be internally validated in order to be merged._

### CODE REVIEW

**Comment by sarneijim** [comment]
Watcher is crying for console.logs :)

File: apps/ledger-live-desktop/src/renderer/drawers/SwapOperationDetails/index.tsx [hunk 1]
```diff
@@ -167,7 +167,8 @@ const SwapOperationDetails = ({
             showCode
             val={normalisedFromAmount}
             fontSize={6}
-            disableRounding
+            // disableRounding
```
[the remark this answers is not in the document]
Comment (ggilchrist-ledger) [reply apps/ledger-live-desktop/src/renderer/drawers/SwapOperationDetails/index.tsx]: Yep you're right. I'll add it to the bug ticket and put it back as before

File: libs/coin-framework/src/currencies/formatCurrencyUnit.ts [hunk 2]
```diff
@@ -141,6 +142,18 @@ export function formatCurrencyUnitFragment(
   });
   frags.splice(nonSepIndex + 1); // remove extra space at the end
 
+  // if (
+  //   frags.some(frag => {
+  //     // frag.value === "ETH";
+  //   })
+  // ) {
+  console.log("FRAGSSSS", frags);
```
[the remark this answers is not in the document]
Comment (ggilchrist-ledger) [reply libs/coin-framework/src/currencies/formatCurrencyUnit.ts:150]: haha not at all, good spot

**Review by valpinkman — APPROVED** [review]

**Pull request merged**